### ISSUE TITLE
PHP 8.1 compatibility

### ISSUE THREAD

**Issue by AlexTr** [unacms/UNA#3828]
1. Error when saving settings in Developer

```
<b>Fatal error</b>:  Uncaught Error: Unknown named parameter $min in /var/www/vhosts/mainsite/html/test/studio/classes/BxDolStudioOptions.php:300
Stack trace:
#0 /var/www/vhosts/mainsite/html/test/studio/template/scripts/BxBaseStudioOptions.php(151): BxDolStudioOptions-&gt;saveChanges()
#1 /var/www/vhosts/mainsite/html/test/studio/options.php(32): BxBaseStudioOptions-&gt;getCode()
#2 {main}
  thrown in <b>/var/www/vhosts/mainsite/html/test/studio/classes/BxDolStudioOptions.php</b> on line <b>300</b><br />
```

**Comment by AlexTr** [comment]
@AntonLV there is an error:
```
Deprecated: Automatic conversion of false to array is deprecated in /var/www/vhosts/mainsite/html/test/modules/base/groups/classes/BxBaseModGroupsFormPrice.php on line <i>36</i>
```
When adding pricing for group based modules

**Comment by AlexTr** [comment]
@AntonLV errors all over in Studio:
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/3324866/175506874-9030ce97-d630-4ce6-90c8-5a399832d959.png">

**Comment by AntonLV** [comment]
Fixed here: [84449e9](https://github.com/unaio/una/commit/84449e953ca28275ae7c579447241aefbeda65e8)
